packages/eas-cli/src/utils/__tests__/profiles-test.ts

```diff
@@ -94,3 +114,89 @@ describe(getProfilesAsync, () => {
     ).rejects.toThrowError(/eas.json is not valid/);
   });
 });
+describe(maybePrintBuildProfileDeprecationWarningsAsync, () => {
+  afterEach(() => {
+    clearHasPrintedDeprecationWarnings();
+    newLineSpy.mockClear();
+    warnSpy.mockClear();
```

Comment:
Can you please check whether this is necessary at all? I think our jest configuration could be clearing mocks by default after every test case.

Reply:
It is needed, they don't get cleared by default